packages/rum/README.md

```diff
@@ -277,7 +282,7 @@ datadogRum.init({
 });
 ```
 
-For example, to collect 25% of your sessions using the Browser RUM option and 25% of your sessions using the Session Replay RUM option, set the `sampleRate` and the `replaySampleRate` to 50:
+For example, to collect only 50% of your overall sessions and the other 25% of your sessions using the Browser RUM option and 25% of your sessions using the Session Replay RUM option, set the `sampleRate` and the `replaySampleRate` to 50.:
```

Comment:
Do we still need the paragraph below the code?

Reply:
I was informed that this section was confusing so I wanted to keep it to make sure it was very clear